ea_dicm2nii.m

```diff
@@ -1,3 +1,17 @@
 function ea_dicm2nii(indir,outdir)
+%
```

Comment:
Please provide a description of this function.

Reply:
Wrapper for dicm2nii (Xiangrui Li, https://github.com/xiangruili/dicm2nii)